### PR TITLE
Hosted service

### PR DESCRIPTION
**Pull request by ThomasBleijendaal**

### CODE REVIEW

**Comment by ThomasBleijendaal** [comment]
Fixes #12

File: Patterns/HostedService.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Hosted Service
+
+When a simple background service is required which can access the same objects in memory of an application, [Hosted Services](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services?view=aspnetcore-5.0&tabs=visual-studio) are a good option. Pro-actively refreshing cache in a `IMemoryCache` is a very good example of when to use Hosted Services.
+
+When not to use Hosted Services:
+
+* The background service does something that is important: Hosted services are not reliable since they run in a Web App on the background. If an instance is taken offline by the Web App load balancer, the hosted service could be terminated mid-process.
+* The background service does not require access to the same objects: It's better to employ a Function App which shares the same App Service Plan than to use a Hosted Service when the background service. Using a seperate App Service Plan further isolates the Function App, which can improve its realiabilty.
```
Comment (Nuss9) [inline Patterns/HostedService.md:8]: Typo in 'reliability'

**Review by Nuss9 — APPROVED** [review]
Only found a typo, otherwise OK. 
Next time, try a rebase instead of a merge, to get up to date with the main branch.

**Comment by ThomasBleijendaal** [comment]
Where do you see that I did a merge? I always rebase, I don't like merging.

**Comment by Nuss9** [comment]
One of the commit titles says 'merge fix'. But that would probably have been a different merge.
You can finish, squash and merge this PR.

**Pull request merged**